### ISSUE TITLE
Missing alt text in SanityImage Component

### ISSUE THREAD

**Issue by LucaArgentieri** [nuxt-modules/sanity#1153]
### Is your feature request related to a problem? Please describe.
I was rendering an image in my page when I noticed that the alt element was not generated.

### Describe the solution you'd like to see
Not sure if you can add a groq query and include the alt in the component or incude all metadata of the image

### Describe alternatives you've considered
The only alternative I found is to do the groq query requesting only url and altText and pass them to an image tag

**Comment by isakgustavsen** [comment]
As the component passes attributes on you can simply add the alt directly. Given that sanity doesn't have alt as a default part of the image you can't query directly for it either

**Comment by isakgustavsen** [comment]
Did not know it was an option there. If you reopen I can have a look. Is the media browser plugin all that is needed?

**Comment by isakgustavsen** [comment]
Having looked at it the extra fields are something you need to query for. The image component doesn't actually query for information, but creates a link based on the ref and a standard url format, meaning it doesn't have a way to get the extra info. 

Unless sanity starts including alt text in their images the best solution is probably the status quo of just adding the property yourself
